docs/en/basic/Configuration-Settings.md

```diff
@@ -179,6 +182,9 @@ To list all of the configuration properties with sources:
 
 ```bash
 ./bin/alluxio getConf --source
+```
```

Comment:
same here, looks like the following is the output and could be in the same block

Reply:
yeah seems to be unnecessary. fixed.

Reply:
fixed.